upgrade/upgrade-scripts/verify-jar

```diff
@@ -42,6 +44,11 @@ function verify_jar_verify_cleanup() {
 			die "failed to cleanup postgres for snapshot '${MDS_SNAPNAME}'."
 	fi
 
+	if [[ -d "$JAR_PATH" ]]; then
```

Comment:
This conditional is required in the event that we trigger the cleanup logic prior to actually installing the package, correct?

If so, I think it'd be better to check to see if the package is installed, rather than checking for that directory.. i.e.
```
if dpkg-query -W delphix-upgrade-verification &>/dev/null; then
    apt_get remove ...
fi
```

IMO, this way it's more clear that we only want to attempt to remove the package, if the package is currently installed.

After this change, we could inline the `JAR_PATH` variable since it'll only be used in one place within this script.